src/main/jbake/content/documentation.adoc

```diff
@@ -0,0 +1,26 @@
+= TomEE Documentation
+:jbake-date: 2016-03-16
+:jbake-type: page
+:jbake-status: published
+:jbake-tomeepdf:
+
+=== Administration
+- link:admin/configuration/index.html[How to configure]
+- link:admin/directory-structure.html[Directory Structure]
+- link:admin/cluster/index.html[Clustering]
+
+=== Developers
+- link:developer/ide/index.html[IDEs - Eclipse, Intellij Idea and Netbeans]
+- link:developer/testing/index.html[Testing - Arquillian, OpenEJB JUnit, TomEE Embedded and ApplicationComposer]
```

Comment:
Unit Testing - Arquillian, OpenEJB JUnit, TomEE Embedded and ApplicationComposer